app/services/session.js

```diff
@@ -0,0 +1,25 @@
+import SessionService from 'ember-simple-auth/services/session';
+import { inject as service } from '@ember/service';
+import ENV from 'frontend-kaleidos/config/environment';
+
+export default class ExtendedSessionService extends SessionService {
+  @service currentSession;
+
+  async handleAuthentication() {
+    try {
+      await this.currentSession.load();
+    } catch (error) { // eslint-disable-line no-unused-vars
+      this.invalidate();
+    }
+    super.handleAuthentication('agendas');
+  }
+
+  handleInvalidation() {
+    const logoutUrl = ENV.torii.providers['acmidm-oauth2'].logoutUrl;
+    if (logoutUrl.startsWith('http')) {
```

Comment:
I this is a validation, it isn't super self-documenting. a `try ... catch` with [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) might be better suited?

Reply:
Indeed. Copied that snippet from lblod/loket :see_no_evil: 
Will change this. `try ... catch` is a better approach and will also handle the case if `logoutUrl` is not configured.